File: simple_reversed_parenthesis/reversed_parenthesis.py
import re

def solve(st):
  if len(st) % 2 != 0:
    return -1

  while '()' in st:
    st = st.replace('()', '')

  new_st = re.sub('\\)\\)|\\(\\(', '', st)
  return (len(st) - len(new_st)) / 2 + len(new_st)

File: simple_reversed_parenthesis_js/reversed_parenthesis.js
function solve(s) {
  if (s.length % 2 !== 0) {
    return -1
  }

  while (s.includes('()')) {
    s = s.replace('()', '')
  }

  l = 0
  for (let i = 0; i < s.length; i++) {
    if (s.charAt(i) == '(') {
      l += 1;
    }
  }

  return Math.ceil(l / 2) + Math.ceil((s.length - l) / 2)
}
module.exports = solve;
